packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -15,6 +15,52 @@ import {
  */
 const Media = {};
 
+Media.localTracks = (audio, video, stream) => {
+  Media.localAudioTracks(audio, stream);
+  Media.localVideoTracks(video, stream);
+};
+
+Media.localAudioTracks = (audio, stream) => {
```

Comment:
Maybe name this `hasLocalAudioTracks`? From the current name it looks like it should return the actual tracks (an array or object) but we are returning a boolean after some manipulation.